data/json/monsters/insect_spider.json

```diff
@@ -1356,5 +1356,34 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "EAT_CROP", 120 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "LARVA", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_mantis",
+    "type": "MONSTER",
+    "name": { "str": "giant preying mantis" },
+    "description": "A preying mantis, the size of a horse. If you weren't looking closely, you might mistake it for a dead tree.",
+    "default_faction": "mantis",
+    "bodytype": "flying insect",
+    "species": [ "INSECT" ],
+    "volume": "550 L",
+    "weight": "275 kg",
+    "hp": 85,
+    "speed": 160,
+    "material": [ "flesh" ],
+    "symbol": "♠",
```

Comment:
You should probably use an ASCII character for the monster symbol, or you risk the monster displaying as a blank box `򪪪` for people using a non-compatible font. 

Aditionally, if you aren't using a tileset, trees look like `7` or `4` so the with the current symbol character is obviously not a tree.

Reply:
Sensible. It kind of feels like a hack to begin with, honestly; what would be better is if it could remain hidden like a trap unless the player's perception skill is high enough. Unfortunately I'm not sure how to implement that. Perhaps just a 'g', like for locusts?

Reply:
Wasn't something similar implemented for shady zombies at some point? They are invisible unless you are standing less than X tiles away, where X depends on your perception stat. Or did it not go through?